mercury_app/views.py

```diff
@@ -167,19 +167,19 @@ def get_context_data(self, **kwargs):
         context = super(SelectEvents, self).get_context_data(**kwargs)
         token = get_auth_token(self.request.user)
         organizations = get_api_organization(token)
-        events = {'events': get_events_for_organizations(
+        events, pagination = get_events_for_organizations(
             organizations,
             self.request.user,
-        )}
-        paginator = Paginator(tuple(events['events']), 10)
-        page = self.request.GET.get('page')
-        try:
-            pagination = paginator.page(page)
-        except PageNotAnInteger:
-            pagination = paginator.page(1)
-        except EmptyPage:
-            pagination = paginator.page(paginator.num_pages)
-        context['pagination'] = pagination
+            self.request.GET.get('page'),
+        )
+        context['events'] = events
+        if pagination:
+            context['has_next'] = pagination.get('has_more_items')
+            context['has_previous'] = True if pagination.get('page_number') > 1 else False
```

Comment:
isn't a little bit the same?
`True if pagination.get('page_number') > 1 else False`
`pagination.get('page_number') > 1`